### PR TITLE
Algorithm and write-up about sorting and insertion sort

### PR DESCRIPTION
**Pull request by wordhou**
- [X] Top-level README “Table of Contents” is updated
 - [X] Feature tasks for this challenge are completed
 - [X] Unit tests written and passing
     - [X] “Happy Path” - Expected outcome
     - [X] Expected failure
     - [X] Edge Case (if applicable/obvious)
 - [X] README for this challenge is complete
     - [X] Summary, Description, Approach & Efficiency, Solution
     - [X] Link to code
     - [X] Picture of whiteboard

### CODE REVIEW

File: docs/sorting.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Day 26: Insertion Sort
+
+## Sorting
```
Comment (Mattpet26) [inline docs/sorting.md:3]: Fantastic work with your readme! Very well written out.
